cmd/dep/init.go

```diff
@@ -350,13 +335,31 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 				// project is not present in the workspace, and so we need to
 				// solve to deal with this dep.
 				r := filepath.Join(ctx.GOPATH, "src", string(pr))
-				_, err := os.Lstat(r)
-				if os.IsNotExist(err) {
+				fi, err := os.Stat(r)
```

Comment:
Because, unless I misunderstand how this all works, we could `Lstat` a symlink that exists, but its referent does not. In that case, the subsequent check of `os.IsNotExist()` would not have the desired effect - it would tell us the symlink exists, but what we really care about is the referent existing.

...At least, I think that's what I _think_ we care about here. My latest explorations into symlinks, and what we should do/the toolchain currently does left me dizzy.